.github/workflows/golangci-lint.yml

```diff
@@ -38,6 +38,12 @@ jobs:
           nix_path: nixpkgs=channel:nixos-unstable
       - name: Lint relayer
         run: nix develop -c make lint-go-relayer
+      - name: Store Golangci lint relayer report artifact
+        if: always()
```

Comment:
Should not be needed if always:
```suggestion
```

Reply:
We still want to get artifacts uploaded, no matter if the linter (a previous step) fails. So `if: (always)` is needed here, like in the rest of other cases.